src/pages/blogpage/BlogPage.jsx

```diff
@@ -0,0 +1,66 @@
+import {Link, useNavigate, useParams} from "react-router-dom";
+
+import './BlogPage.css'
+import TimeDate from '../../helpers/TimeDate.js'
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {CaretLeft, Clock} from "@phosphor-icons/react";
+
+
+
+function BlogPage() {
+    const { id } = useParams();
+
+
+    const [oneBlog,setOneBlog]=useState([])
+    const [error,setError]=useState()
+    const navigate = useNavigate()
+
+    async function fetchOneBlog() {
+        try {
+            const result = await axios.get(url)
+            console.log(result.data)
+            setOneBlog(result.data)
+
+        } catch (e) {
+            setError(e)
+            console.error(error);
+        }
+    }
+
+useEffect(() => {
+        fetchOneBlog()
+    }, []);
+    const url ='http://localhost:3000/posts/' +id
+
+
+
+
+    return (
+        <>{Object.keys(oneBlog).length > 0 ?
```

Comment:
Hier wordt 'ie dan wat simpeler
```suggestion
        <>{oneBlog ?
```